src/SharedPipelineComponents/IlrSubmissionEvents/SFA.DAS.Provider.Events.Submission.UnitTests/SubmissionEventsProcessorTests/WhenProcessing.cs

```diff
@@ -156,12 +171,16 @@ public void ThenItShouldWriteAnEventForEachIlrDeliveryThatHasChanged()
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].ProgrammeType == null)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].FrameworkCode == null)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].PathwayCode == null)));
-            _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].ActualStartDate == null)));
+            _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].ActualStartDate == updatedSubmissionChanged.ActualStartDate)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].PlannedEndDate == updatedSubmissionChanged.PlannedEndDate)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].ActualEndDate == null)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].OnProgrammeTotalPrice == updatedSubmissionChanged.OnProgrammeTotalPrice)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].CompletionTotalPrice == updatedSubmissionChanged.CompletionTotalPrice)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].NiNumber == null)));
+            _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].EPAOrgId == ilrForFirstSubmission.EPAOrgId)));
```

Comment:
This check already exists below.

Reply:
Yeah, looks like a merge issue.  Fixed. 👍